storybookjs/storybook
# Addon-docs: Module story format & framework param

Issue: #7101 

## What I did

This PR combines two different ideas: [module story format](https://gist.github.com/tmeasday/4de20eab47226a870ab1025642ba848c) and adding a "framework" parameter for stories. I combined the two PRs because they touch the same code and it's a pretty simple "regular" change that gets repeated across all the apps.

```js
export default {
  title: 'Root/path/to/component'
}
export const story1 = () => <Component />
export const story2 = () => <Component something={false} />
```

## How to test

```
cd examples/official-storybook
yarn storybook
```

Probably should add unit tests as well, and potentially convert the other examples to the new load API. Suggestions?
